test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
So presumably, the line that's failing looks like:

```js
speech.recognize({
  contents: fs.readFileSync('C:\Users\\appveyor\\AppData\\Local\\Temp\\mp-2676Xj4IRwkvTWvY.tmp')
}, ...
```

I think we might need to use `C:\\`, which can be done by calling `path.normalize(temporaryFile.name)`